tensorflow/python/ops/math_ops.py

```diff
@@ -4237,6 +4237,25 @@ def polyval(coeffs, x, name=None):
   Returns:
     A `tensor` of the shape as the expression p(x) with usual broadcasting
     rules for element-wise addition and multiplication applied.
+    
+  Usage Example:
```

Comment:
Hey, everything looks good, just that the Usage Example should be placed above the 'Args' section, and trailing whitespace should be trimmed.

Reply:
Thanks for reviewing! I've made the changes.